Tests/PointFreeTests/EpisodeTests.swift

```diff
@@ -398,19 +398,24 @@ class EpisodeTests: TestCase {
       from: request(to: .episode(.left(Current.episodes().first!.slug)), session: .loggedIn)
     )
 
-    assertSnapshot(matching: conn |> siteMiddleware, as: .ioConn)
 
     #if !os(Linux)
     if #available(OSX 10.13, *), ProcessInfo.processInfo.environment["CIRCLECI"] == nil {
-      assertSnapshots(
-        matching: conn |> siteMiddleware,
-        as: [
-          "desktop": .ioConnWebView(size: .init(width: 1100, height: 1600)),
-          "mobile": .ioConnWebView(size: .init(width: 500, height: 1900))
-        ]
-      )
+      let webView = WKWebView(frame: .init(x: 0, y: 0, width: 1100, height: 1600))
```

Comment:
creating this web view explicitly so that I can execute JS in it for opening the details of one of the exercises.